src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -111,4 +111,8 @@ public void resetPose(Pose2d pose) {
   public Command resetGyroCommand(){
     return swerve.zeroGyroCommand();
   }
+  
+  public void targetAngleDrive(Rotation2d moduleDirection) {
+    swerve.targetAngleDrive(moduleDirection);
```

Comment:
The passed value should be called `targetAngle`. We don't want to set the module rotations directly. We want to set the robot's direction. The modules will spin as needed by the control method to point the robot in the right direction